src/Parts/FilterTrainingEvent/index.jsx

```diff
@@ -26,13 +26,17 @@ const FilterTrainingEvent = () => {
 
   const onChangeSearching = value => {
     // setValueCardTraining(value);
```

Comment:
remove unused line

Reply:
thankyou for review, this issue already fix on 4da229e93e8066f1e4b3e33ea8e2a40e32f9c6d0